DiscordBot/Backends/Discord/DiscordSettings.cs

```diff
@@ -0,0 +1,28 @@
+using System.Collections.Generic;
+using DSharpPlus.EventArgs;
+using TicTacToe;
+
+namespace DiscordBot.Backends.Discord
+{
+    public class DiscordSettings

```

Comment:
If both Settings classes have the same interface, then...

Reply:
i tried but it didn't work at the start
don't remeber if i kept the classes' variables static, if so, it doesn't work with that
if i didn't, then you're right i should do a backend